src/transformers/models/fnet/configuration_fnet.py

```diff
@@ -0,0 +1,113 @@
+# coding=utf-8
+# Copyright Google Research and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" FNet model configuration """
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+FNET_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "gchhablani/fnet-base": "https://huggingface.co/gchhablani/fnet-base/resolve/main/config.json",
```

Comment:
Let's maybe move this to the google organization no? @gchhablani - I can add you to the org, so that you can upload models under `google/...` :-)

Reply:
@patrickvonplaten Yes, that is just there temporarily for tests :p